aspnetcore/fundamentals/openapi/aspnetcore-openapi.md

```diff
@@ -255,6 +257,8 @@ var app = builder.Build();
 + app.MapOpenApi();
 ```
 
+:::moniker end
```

Comment:
```suggestion
```

Reply:
I've made changes, see the other comment. i'll resolve this conversation.